lesson9/task1/Main.java

```diff
@@ -0,0 +1,26 @@
+package task1;
+
+import java.io.*;
+
+public class Main {
+    static boolean isPalindrome(String word) {
+        return word.equals(new StringBuilder(word).reverse().toString());
+    }
+    public static void main(String[] args) throws IOException {
+        File fileOutput = new File("D:\\TMS\\Lesson9Home\\src\\task1\\task1_file2.txt");
+        try (var fileWriter = new FileWriter(fileOutput)) {
```

Comment:
Поток чтения надо тоже закрывать корректно. И поток записи лучше тоже буфферизировать.
```suggestion
        try (var fileWriter = new BufferedWriter(new FileWriter(fileOutput));
        var inputPalindrome = new BufferedReader( new FileReader(fileInput))
        ) {
```